.github/workflows/continuous-integration.yml

```diff
@@ -1,81 +0,0 @@
-name: Continuous Integration
```

Comment:
why don't we need this anymore?

Reply:
In this branch I am only using PHP 8. Did not have the time to review this file just for this branch :smile:

Reply:
ah i see, it's a throwaway-branch